### PR TITLE
adds publishing rules for kube 1.16.0-rc.2

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

File: publishing-rules.yaml [hunk 1]
```diff
@@ -3,6 +3,10 @@ skip-gomod: true
 rules:
 - destination: kubernetes
   branches:
+  - name: origin-4.3-kubernetes-1.16.0-rc.2
```
Comment (p0lyn0mial) [inline publishing-rules.yaml]: is this the name of the branch? - https://github.com/openshift/kubernetes/tree/origin-4.3-kubernetes-1.16.0-rc.2
where do I find definitions for the rules?

**Comment by deads2k** [comment]
hmmm... network-edge went sideways?

/retest

**Pull request merged**